blackjack/console.py

```diff
@@ -0,0 +1,155 @@
+from core import GameState, render_hand, render_dealer, Conclusion, hand_total, parse_play, Play, make_deck, \
+    initial_deal, has_blackjack, hit, get_winner, check_split, check_deck, check_split_response, split_hand, set_table,\
+    \
+    check_for_bust
```

Comment:
You have an unnecessary `\` on line 3. However, I think you can use parentheses to clean up this import, e.g.

```python
from core import (
  Conclusion, GameState, Play, ...
)
```

Also, in PyCharm, there should be a "re-format imports" action that should clean this up for you. Good to see what it does so you understand.